packages/server/middlewares/xssValidation.ts

```diff
@@ -0,0 +1,18 @@
+import type { NextFunction, Request, Response } from 'express';
+import { body, validationResult } from 'express-validator';
+
+export const xssValidator = () =>
+  body('*').not().contains('</script>', { ignoreCase: true });
```

Comment:
Лучше использовать готовую утилиту типо https://github.com/leizongmin/js-xss

Reply:
И кстати у body уже есть санитайзер, https://express-validator.github.io/docs/guides/getting-started#sanitizing-inputs
```suggestion
  body('*').escape();
```